10.u-root/README.md

```diff
@@ -4,6 +4,45 @@ Points of contact: [Ronald Minnich](https://github.com/rminnich)
 
 This chapter describes u-root, what it is, and why it is the way it is.
 
+## Quick Start
+
+Build a u-root initramfs in busybox mode.
+
+	$ go run github.com/u-root/u-root -build=bb
+	2019/02/11 18:29:00 Build environment: GOARCH=amd64 GOOS=linux GOROOT=/home/rminnich/go GOPATH=/home/rminnich/gopath CGO_ENABLED=0
+	2019/02/11 18:29:00 Filename is /tmp/initramfs.linux_amd64.cpio
+	2019/02/11 18:29:12 Successfully wrote initramfs.
+
+This command scans all the u-root source code with the GO AST package, converting commands into packages, and compiling them into one binary. It takes around fifteen seconds.  It creates a file called /tmp/initramfs.linux_amd64.cpio. To test it, you need to install the testramfs command. 
```

Comment:
nit: you could briefly specify that this process saves a large amount of space, proportional to the number of programs

Reply:
done